src/core/utils.js

```diff
@@ -1,17 +1,3 @@
 'use strict'
 
-exports.ifRepoExists = (repo, cb) => {
-  repo.exists((err, exists) => {
-    if (err) {
-      return cb(err)
-    }
-
-    if (exists === false) {
-      return cb(new Error('no ipfs repo found'))
-    }
-
-    cb()
-  })
-}
-
```

Comment:
Ah, not on the readme, people are using it though - https://github.com/ipfs/js-ipfs/pull/776

Wanna finish that PR and merge it here?